docs/content/posts/2024-03-xx-release-1.19.0.md

```diff
@@ -0,0 +1,456 @@
+---
+authors:
+- LincolnLee:
+  name: "Lincoln Lee"
+  twitter: lincoln_86xy
+
+date: "2024-03-xxT22:00:00Z"
+subtitle: ""
+title: Announcing the Release of Apache Flink 1.19
+aliases:
+- /news/2024/03/xx/release-1.19.0.html
+---
+
+The Apache Flink PMC is pleased to announce the release of Apache Flink 1.19.0. As usual, we are
+looking at a packed release with a wide variety of improvements and new features. Overall, 162
+people contributed to this release completing 33 FLIPS and 600+ issues. Thank you!
+
+Let's dive into the highlights.
+
+# Flink SQL Improvements
+
+## Custom Parallelism for Table/SQL Sources
+
+Now in Flink 1.19, you can set a custom parallelism for performance tuning via the `scan.parallelism`
+option. The first available connector is DataGen (Kafka connector is on the way). Here is an example
+using SQL Client:
+
+```sql
+-- set parallelism within the ddl
+CREATE TABLE Orders (
+    order_number BIGINT,
+    price        DECIMAL(32,2),
+    buyer        ROW<first_name STRING, last_name STRING>,
+    order_time   TIMESTAMP(3)
+) WITH (
+    'connector' = 'datagen',
+    'scan.parallelism' = '4'
+);
+
+-- or set parallelism via dynamic table option
+SELECT * FROM Orders /*+ OPTIONS('scan.parallelism'='4') */;
```

Comment:
I've checked the last few release announcements and we've included some simple examples in some of the feature introductions since 1.18(https://flink.apache.org/2023/10/24/announcing-the-release-of-apache-flink-1.18/), which from a user's point of view is probably more of an introduction to what a feature looks like, rather than a complete description of how to use the feature.
I'm not sure how to choose between simplicity and readability, maybe we need to hear more feedback on this, or discuss it in the next release sync and I'll change it accordingly : )